CONTRIBUTORS_GUIDE.md

```diff
@@ -0,0 +1,108 @@
+# 🌟 Contributing Guide | PayStell
+
+We’re excited you’re interested in contributing! Please follow these steps to ensure a smooth and efficient collaboration process.
+
+🎉 Thank you for being interested in contributing to the PayStell project! 🎉
+
+Feel welcome and read the following sections in order to know how to ask questions and how to work on something.
+
+Please make sure you are welcoming and friendly in all of our spaces.
+
+We're really glad you're reading this, because we need volunteer developers to help this project come to fruition. 👏
+
+## 👨🏻‍💻 Issues
+
+The best way to contribute to our projects is to apply to the issues <a href="https://github.com/PayStell/paystell-website/issues" target="_blank">here</a> .
+
```

Comment:
add: PRs are not allowed; all PRs must originate from an assigned issue.

Reply:
Okay, I'm going to work on that.